### PR TITLE
Fix 818 id name

### PR DESCRIPTION
**Pull request by micahwoodard**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions on the [wiki](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki)
- If computer maintainers need to manually update anything, provide detailed step by step instructions
- Use markdown syntax in order for your comments to be rendered reliably in the email: "1." instead of "1)", use four spaces for indents.
- If you use the keyword "skip email" in the title, it will skip the email updates
- Merges from "develop" into "production_testing" should use the keyword "production merge" in the title for reliable indexing of updates
- Merges from "production_testing" into "main" should use the keyword "update main"
  
### Describe changes:
- fix mismatch between mouse id and experimenter name
### What issues or discussions does this update address?
- resolves [#818](https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/818)
### Describe the expected change in behavior from the perspective of the experimenter
- name and id should match in load drop down 
### Describe any manual update steps for task computers
- None
### Was this update tested in 446/447?
- [x] 446/7

### CODE REVIEW

**Comment by micahwoodard** [comment]
@alexpiet @XX-Yin could we also merge this into production_testing so we can role out updates this week? I feel like merging production_testing with this issue unresolved still lingering could be confusing and cause issues.

File: src/foraging_gui/Foraging.py [hunk 1]
```diff
@@ -2987,7 +2987,9 @@ def _Open(self,open_last = False,input_file = ''):
                 # list of mice, experimenters, and two week in chronological order form date modified
                 mice, experimenters, two_week = self._Open_getListOfMice()
                 # only add mice from two weeks in drop down.
-                W = MouseSelectorDialog(self, [m + ' ' + n for m, n in zip(two_week, experimenters)])
+                print(mice)
```
Comment (XX-Yin) [inline src/foraging_gui/Foraging.py:2990]: Why print(mice) and print(two_week) here?

**Comment by XX-Yin** [comment]
> @alexpiet @XX-Yin could we also merge this into production_testing so we can role out updates this week? I feel like merging production_testing with this issue unresolved still lingering could be confusing and cause issues.

I'm fine with making a parallel merging to the production_testing

File: src/foraging_gui/Foraging.py (hunk 1 above)
Comment (micahwoodard) [reply src/foraging_gui/Foraging.py:2990]: I need to do one more test in 446 but integration testing is going on. I will take out before merging!

**Review by alexpiet — APPROVED** [review]
Yes, sorry I missed this

**Review by XX-Yin — APPROVED** [review]

**Pull request merged**